MaxText/configs/base.yml

```diff
@@ -61,6 +61,13 @@ save_config_to_gcs: False
 dtype: "bfloat16"
 quantization: "" #defaults to no quantization, i.e. bf16. possible alternative setting is 'int8' or use fp8 to run with 8-bit floating-point GeMMs on NVIDIA GPUs.
 quantize_kvcache: False
+# Valid kv_quant_axis values:
+#   - "" is valid only when quantize_kvcache is False
+#   - "dkv" over kv_dimensions
+#   - "heads_and_dkv" over kv_heads and kv_dimensions
+# Default to "heads_and_dkv" for faster compution, any valid kv_quant_axis non "" value will not be used when quantize_kvcache is False
```

Comment:
Nit: Reword 

any valid kv_quant_axis non "" value will not be used when quantize_kvcache is False

to

kv_quant_axis is not used when quantize_kvcache is False